### PR TITLE
Automated lifecycle-scaling test

### PR DESCRIPTION
**Pull request by shaior**

### CODE REVIEW

**Review by greyerof — APPROVED** [review]

File: tests/lifecycle/tests/lifecycle_scaling.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifehelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/lifecycle/lifeparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+// in order to run the test, export TNF_NON_INTRUSIVE_ONLY=false.
```
Comment (kononovn) [inline tests/lifecycle/tests/lifecycle_scaling.go:13]: Please set this variable automatically in your test and then remove it in After each section. Use  os.Setenv func.

**Review by kononovn — CHANGES_REQUESTED** [review]
Looks good. Thanks you for keeping your PRs small. Just 1 more minor change required.

**Comment by kononovn** [comment]
Please add 1 more suite in README file

**Review by kononovn — APPROVED** [review]
Looks good to me

**Pull request merged**